pkg/printers/customcolumn_flags.go

```diff
@@ -84,7 +84,7 @@ func (f *CustomColumnsPrintFlags) ToPrinter(templateFormat string) (ResourcePrin
 	}
 
 	p, err := NewCustomColumnsPrinterFromSpec(templateValue, decoder, f.NoHeaders)
-	return NewVersionedPrinter(p, legacyscheme.Scheme, legacyscheme.Scheme, scheme.Versions...), err
+	return NewVersionedPrinter(p, legacyscheme.Scheme, legacyscheme.Scheme, scheme.Scheme.PreferredVersionsAllGroups()...), err
```

Comment:
all `Versions...` should be replaced with `PrioritizedVersionsAllGroups()...` to remain equivalent, right?